client/src/components/Header.tsx

```diff
@@ -442,36 +457,38 @@ function Header() {
             </>
           ) : (
             <>
-              <NavLink className={classes.navLink} to={routes.Signup.path}>
-                <Button
-                  sx={{
-                    textTransform: 'capitalize',
-                    backgroundColor: '#EF6A60',
-                    color: 'white',
-                    borderRadius: '10px',
-                    border: '1px solid #EF6A60',
-                    marginRight: '10px',
-                    width: '100px',
-                  }}
-                >
-                  Join Now
-                </Button>
-              </NavLink>
-              <NavLink className={classes.navLink} to={routes.Login.path}>
-                <Button
-                  sx={{
-                    textTransform: 'capitalize',
-                    backgroundColor: 'white',
-                    color: '#323232',
-                    borderRadius: '10px',
-                    border: '1px solid #323232',
-                    marginLeft: '5px',
-                    width: '100px',
-                  }}
-                >
-                  Sign In
-                </Button>
-              </NavLink>
+              {/* <NavLink className={classes.navLink} to={routes.Signup.path}> */}
+              <Button
+                sx={{
+                  textTransform: 'capitalize',
+                  backgroundColor: '#EF6A60',
+                  color: 'white',
+                  borderRadius: '10px',
+                  border: '1px solid #EF6A60',
+                  marginRight: '10px',
+                  width: '100px',
+                }}
+                onClick={() => handleoOpenModal('SignUp')}
+              >
+                Join Now
+              </Button>
```

Comment:
Here, we should be using the color from the theme.

Reply:
Moved to styles and used all of the theme.palette colors